clang/test/CodeGenSYCL/debug-info-checksum-temp-name.cpp

```diff
@@ -24,7 +24,6 @@
 //
 // CHECK: !DICompileUnit({{.*}} file: ![[#FILE:]]
 // CHECK: ![[#FILE]] = !DIFile(filename: "{{.*}}clang{{.+}}test{{.+}}CodeGenSYCL{{.+}}Inputs{{.+}}debug-info-checksum.cpp"
-// CHECK-SAME: directory: "{{.*}}clang{{.+}}test{{.+}}CodeGenSYCL{{.+}}Inputs"
```

Comment:
Should we check the right directory instead?

Reply:
The directory is the test directory. It can be any /tmp folder where we are running the test. It can't be represented with a regular expression.

Reply:
For some reason it seems that it is important that debug info also contains path to the directory where the test is running, right? So, I'm just asking for a test to make sure we have what we want and not break this in the future.
If the info about where the test is running is not important, why do we need the patch then?

Reply:
The value of the directory is the directory from where the test is ran. This can't be represented by %S. %S corresponds to "../test/CodeGenSYCL (where the test resides).
If you look at the history of this patch, you will see that the directory didn't appear in the LIT tests. I have added them in my last patch, thinking that the directory was the root of the test case. But that's not the case. This now in sync with community.

Reply:
Okay, this makes sense. It seems the community also doesn't have test for that.